packages/api/src/hooks/useNotificationEvent.ts

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import useRequest from '../useRequest';
+import useInvalidateQuery from '../useInvalidateQuery';
+import { TSocketRequestPayload } from '../../types';
+
+type TNotificationEventPayload = TSocketRequestPayload<'notification_event'>['payload'];
+
+/**
+ * Hook to send notification event to the server
+ * @name useNotificationEvent
+ * @returns response, mutation function and other properties from useRequest hook
+ */
+const useNotificationEvent = () => {
+    const invalidate = useInvalidateQuery();
+    const { data, mutate, ...rest } = useRequest('notification_event', {
+        onSuccess: () => {
+            invalidate('notification_event');
+        },
+    });
+
+    /**
+     * Function to send notification event to the server
+     * @param payload - notification event payload
+     */
+    const send = React.useCallback((payload: TNotificationEventPayload) => mutate({ payload }), [mutate]);
+
+    return {
+        notification_event_status: data?.notification_event,
```

Comment:
it would be confusing to have this name as useRequest has status which is related to request status
Maybe we can rename it to `data` it self

you can ignore this suggestion as well 🙏

Reply:
I did think of this. But when we have more than one hook in a component, the data name aliasing becomes confusing and may be redundant, so I thought of renaming it in the hook itself. Do u suggest any other name?

Reply:
We can name this `data` and inside the consuming component we can rename the like `data: notification_event` something related